configuration/contrib/diag-plugins/01_tedge.sh

```diff
@@ -46,6 +51,17 @@ collect() {
             collect_logs "tedge-mapper-${cloud}"
         fi
     done
+
+    # Collectd mapper log
+    if command -V collectd >/dev/null 2>&1; then
```

Comment:
Not sure if this condition is the best. Alternatively, the condition can be as below.

```sh
if command -v systemctl >/dev/null 2>&1; then
    if systemctl is-active --quiet tedge-collectd-mapper.service; then
        echo "active"
    fi
fi
```

Reply:
You can simplify the logic and just use the `systemctl list-unit-files` to check both if the systemctl command exists, and the service exists

```sh
if systemctl list-unit-files tedge-collectd-mapper.service  >/dev/null 2>&1; then
    collect_logs tedge-mapper-collectd
fi
```